auto-complete.js

```diff
@@ -171,14 +174,32 @@
                 removeQueryFromLocalStorage(o.queryHistoryStorageName, rawData[index]);
             }, that.sc);
 
-            that.blurHandler = function () {
-                try { var over_sb = document.querySelector('.autocomplete-suggestions:hover'); } catch (e) { var over_sb = 0; }
-                if (!over_sb) {
-                    that.last_val = that.value;
-                    that.sc.style.display = 'none';
-                    setTimeout(function () { that.sc.style.display = 'none'; }, 350); // hide suggestions on fast input
-                } else if (that !== document.activeElement) setTimeout(function () { that.focus(); }, 20);
+            if (o.isMobile) {
+                live('autocomplete-suggestions', 'scroll', function(e) {
+                    if (!that.sc.className.includes(suggestionsFocusClass)) {
+                        that.sc.className += ` ${suggestionsFocusClass}`;
+                    }
+
+                    that.sc.focus();
+                }, that.sc);
+            }
+
+            that.blurHandler = function (e) {
+                var isSuggestionsFocused = document.querySelector('.autocomplete-suggestions--focused');
+                if (!isSuggestionsFocused) {
+                    try { var over_sb = document.querySelector('.autocomplete-suggestions:hover'); } catch (e) { var over_sb = 0; }
```

Comment:
`querySelector` returns `null` if element doesn't exist, use `camelCase`.
What does `Sb` mean?
```suggestion
                    var overSb = document.querySelector('.autocomplete-suggestions:hover');
```

Reply:
It's strange, I prefer to not change the current codebase and use another patch to refactor entire code, including standard os JS used: lint, es6, etc

Reply:
I think Sb means suggestions body, I really don't know 😭